test/fixtures/vm/vm_display_syntax_error.snapshot

```diff
@@ -2,18 +2,20 @@ beginning
 foo.vm:1
 var 4;
     ^
+
 SyntaxError: Unexpected number
     at new Script (node:vm:*)
     at createScript (node:vm:*)
     at Object.runInThisContext (node:vm:*)
-    at Object.<anonymous> (*test*message*vm_display_syntax_error.js:*)
+    at Object.<anonymous> (*fixtures*vm*vm_display_syntax_error.js:31:6)
```

Comment:
Why are these now exact line matches?  I am running into an error locally with this test and many others and it seems the matching of the line numbers are quite arbitrary here. I am surprised that some of them demand exact line/column numbers (that is, `*` cannot match numbers). It also seems the new format is unable to match arbitrary lines, which makes the snapshots unnecessarily brittle compared to the original format..

Reply:
> Why are these now exact line matches?

I don’t think that was intentional, and I think the snapshot creation tool has been improved since this PR to avoid this unintentional precision. We should update the snapshots to remove these precise row/column numbers.